src/test/kotlin/com/igorwojda/list/search/linearsearch/challenge.kt

```diff
@@ -4,7 +4,12 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun getIndex(list: List<String>, str: String): Int {
-    TODO("not implemented")
+    list.forEachIndexed { index, s ->
```

Comment:
Foarte bine. In real life poti folosi direct .indexOf

Reply:
> Foarte bine. In real life poti folosi direct .indexOf

oh wow , clar ceva era in neregula cu mine in momentul in care rezolvam asta de nu m-am gandit la functia aia